### PR TITLE
You have an issue when keycloak start in ha mode, the cli doesn't mod…

### PR DESCRIPTION
**Pull request by tdemaret**
…ify the correct xml file

### What does this PR do?
Just modify the truststore modification when you have a self sign certificate.

### What issues does this PR fix or reference?
No PR actually

### CODE REVIEW

**Comment by skabashnyuk** [comment]
@tdemaret  thanks for your contribution. Can you describe the problem that you are trying to solve?

**Comment by tdemaret** [comment]
Yes of course @skabashnyuk, when I try to deploy Che with Keycloak for multi user use case, keycloak start with standalone-ha configuration : 

1000520+    346  0.9  1.5 2359520 515656 ?      Sl   Dec05   9:57 /usr/lib/jvm/java/bin/java -D[Standalone] -server -Xms64m -Xmx512m -XX:MetaspaceSize=96M -XX:MaxMetaspaceSize=256m -Djava.net.preferIPv4Stack=true -Djboss.modules.system.pkgs=org.jboss.byteman -Djava.awt.headless=true -Dorg.jboss.boot.log.file=/opt/jboss/keycloak/standalone/log/server.log -Dlogging.configuration=file:/opt/jboss/keycloak/standalone/configuration/logging.properties -jar /opt/jboss/keycloak/jboss-modules.jar -mp /opt/jboss/keycloak/modules org.jboss.as.standalone -Djboss.home.dir=/opt/jboss/keycloak -Djboss.server.base.dir=/opt/jboss/keycloak/standalone -Djboss.bind.address=10.128.5.254 -Djboss.bind.address.private=10.128.5.254 -c=standalone-ha.xml -Dkeycloak.migration.action=import -Dkeycloak.migration.provider=dir -Dkeycloak.migration.strategy=IGNORE_EXISTING -Dkeycloak.migration.dir=/scripts/ -Djboss.bind.address=0.0.0.0 

But when you have a Openshift master with self sign certificate the script use the trust store just for standalone.xml. 

This PR is just to modify the correct file for my use case.

Regards,

**Comment by tdemaret** [comment]
No News ? I am alone with this issue ?

**Comment by skabashnyuk** [comment]
CC @tolusha

**Comment by tolusha** [comment]
@tdemaret 
Thank you for your contribution. 
It is a new area of responsibility for me, I will try to validate the PR soon.

**Comment by tolusha** [comment]
@tdemaret 
You have to sign ECA [1] to be have PR merged.
[1] https://www.eclipse.org/legal/ECA.php

File: dockerfiles/keycloak/kc_realm_user.sh [hunk 1]
```diff
@@ -38,6 +38,7 @@ if [ "${CHE_SELF__SIGNED__CERT}" != "" ]; then
     keytool -importcert -alias HOSTDOMAIN -keystore /scripts/openshift.jks -file /scripts/openshift.cer -storepass openshift -noprompt
     keytool -importkeystore -srckeystore $JAVA_HOME/jre/lib/security/cacerts -destkeystore /scripts/openshift.jks -srcstorepass changeit -deststorepass openshift
     /opt/jboss/keycloak/bin/jboss-cli.sh --file=/scripts/cli/add_openshift_certificate.cli && rm -rf /opt/jboss/keycloak/standalone/configuration/standalone_xml_history
+    /opt/jboss/keycloak/bin/jboss-cli.sh --file=/scripts/cli/add_openshift_certificate-ha.cli
```
Comment (tolusha) [inline dockerfiles/keycloak/kc_realm_user.sh:41]: I am not sure about this.
Should we use either `add_openshift_certificate.cli ` or `add_openshift_certificate-ha.cli ` ?

Comment (tdemaret) [reply dockerfiles/keycloak/kc_realm_user.sh:41]: I just add my part for my use case (Openshift deployment / Oauth with Openshift / Multi User / Self Sign Certificate) , and I am not sure if I remove the standalone.xml modification, I don't broke something. I sign ECA

I prefer add my part and modify the standalone-ha.xml with the correct trust store, I don t see all use cases.

**Pull request closed without merging**